packages/@glimmer/integration-tests/test/updating-test.ts

```diff
@@ -125,7 +125,7 @@ class UpdatingTest extends RenderTest {
 
     this.assertHTML(stripTight`
       <div>
-        [empty string]
+        []
```

Comment:
this is different behavior.  Not sure if technically a regression though.  Neither `this.[]` nor `this['']` work.